rpcs3/Emu/Cell/lv2/sys_timer.cpp

```diff
@@ -290,6 +290,10 @@ error_code sys_timer_sleep(ppu_thread& ppu, u32 sleep_time)
 
 error_code sys_timer_usleep(ppu_thread& ppu, u64 sleep_time)
 {
+	if (sleep_time <= 300) {
```

Comment:
return news line here for "{"

Reply:
Thanks.  You can hold off on the code-review -- it won't go in until there is more discussion and code changes.

Reply:
And testing. :-)